ush/soca/marine_recenter.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+
+import os
+from logging import getLogger
+from typing import Dict, List, Any
+from wxflow import logit
+
+logger = getLogger(__name__.split('.')[-1])
+
+
+class MarineRecenter():
```

Comment:
```suggestion
class MarineRecenter(Task):
```

We should base this class on the `Task` class in `wxflow`.  This will give you access to runtime variables such as `PDY`, `cyc`, `RUN`, and more.

```python3
from wxflow import Task
```

Reply:
done